financialaid/api_test.py

```diff
@@ -41,7 +40,9 @@ class FinancialAidBaseTestCase(TestCase):
     @classmethod
     def setUpTestData(cls):
         with mute_signals(post_save):
-            cls.profile = ProfileFactory.create()
+            cls.profile = ProfileFactory.create(
+                country=FuzzyText(length=2)
```

Comment:
if `country` can contain only the country codes, maybe it is better to change the `ProfileFactory` directly.